Seems `resp.space_name() == ''` is expected in current master/nightly-build of NebulaGraph core?

```python
    def test_2_reconnect(self):
        try:
            session = self.pool.get_session('root', 'nebula')
            time.sleep(2)
    
            # wait for the session space info to be updated to meta service
            session.execute(
                'CREATE SPACE IF NOT EXISTS test_session(vid_type=FIXED_STRING(8)); USE test_session;'
            )
            time.sleep(10)
            for i in range(0, 5):
                if i == 3:
                    os.system('docker stop tests_graphd0_1')
                    os.system('docker stop tests_graphd1_1')
                    time.sleep(3)
                resp = session.execute('SHOW SESSIONS')
                assert resp.is_succeeded(), resp.error_msg()
                assert resp.space_name() == 'test_session' #<---------------------
                time.sleep(2)
            session.release()
            new_session = self.pool.get_session('root', 'nebula')
            new_session.execute('SHOW SPACES')
        except Exception as e:
>           assert False, e
E           AssertionError: AssertionError("assert '' == 'test_session' #-----------
E               - test_session") 
E           assert False

```

If that's the case, let's remove this assert from UT.

--- a/tests/test_pool.py
+++ b/tests/test_pool.py
@@ -170,7 +170,7 @@ def test_multi_thread():
     # Test multi thread
     addresses = [('127.0.0.1', 9669), ('127.0.0.1', 9670)]
     configs = Config()
-    thread_num = 50
+    thread_num = 6
     configs.max_connection_pool_size = thread_num
     pool = ConnectionPool()
     assert pool.init(addresses, configs)

--- a/tests/test_session.py
+++ b/tests/test_session.py
@@ -48,9 +48,10 @@ class TestSession(TestCase):
             time.sleep(2)
 
             # wait for the session space info to be updated to meta service
-            session.execute(
+            resp = session.execute(
                 'CREATE SPACE IF NOT EXISTS test_session(vid_type=FIXED_STRING(8)); USE test_session;'
             )
+            assert resp.is_succeeded(), resp.error_msg()
             time.sleep(10)
             for i in range(0, 5):
                 if i == 3:

